cfg/filebackend_test.go

```diff
@@ -0,0 +1,84 @@
+package cfg
+
+import (
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestFileLoad(t *testing.T) {
+	u, _ := url.Parse("file://foobar")
+	backend := NewFileBackend()
+
+	_, err := backend.Load(u)
+	if err != nil {
+		t.Error("Loading an non-existing config file should not return an error")
+	}
+
+	// try to load the config from a relative path
+	u, err = url.Parse(filepath.Join("testdata", "beehive.conf"))
```

Comment:
Doesn't this require `testdata/beehive.conf` to be added to the repo?

Reply:
Already there: https://github.com/muesli/beehive/tree/master/cfg/testdata

That what you mean?